### ISSUE TITLE
YouTube Music Authentication fails unless cookies from Chrome used

### ISSUE THREAD

**Issue by Nikey646** [FoxxMD/multi-scrobbler#98]
**Describe the bug**
Unable to log in to YouTube Music anymore.

**To Reproduce**
1. Try to log in to the YouTube Music Source.

**Expected behavior**
To have the source working.

**Logs**
If possible reproduce the issue with [debug logging ON](https://github.com/FoxxMD/multi-scrobbler/blob/master/docsite/docs/FAQ.md#turn-on-debug-logging)

```
2023-09-16T17:21:07+10:00 error : [Sources] [Ytmusic - MyYTMusic] Cannot start polling because source is not authenticated correctly.
2023-09-16T17:21:07+10:00 info : [App] Scheduler started.
2023-09-16T17:21:05+10:00 warn : [Sources] ytmusic (MyYTMusic) source auth failed.
2023-09-16T17:21:05+10:00 debug : [Sources] Checking ytmusic (MyYTMusic) source auth...
2023-09-16T17:21:05+10:00 info : [Sources] ytmusic (MyYTMusic) source initialized
2023-09-16T17:21:05+10:00 debug : [Sources] Attempting ytmusic (MyYTMusic) initialization...
2023-09-16T17:21:05+10:00 debug : [Sources] (MyYTMusic) Constructing ytmusic source
```

**Versions (please complete the following information):**
Provide version information for any related sources/clients.

- multi-scrobbler: [Updated on 16/9/23]

**Comment by FoxxMD** [comment]
Unfortunately there's nothing I can currently do about this.

The [NPM package at 1.6.0](https://www.npmjs.com/package/youtube-music-ts-api/v/1.6.0) of the base library [youtube-music-ts-api](https://github.com/nickp10/youtube-music-ts-api) does indeed make cookies retrieved from FF work. However, building the library from source at the [1.6.0 commit](https://github.com/nickp10/youtube-music-ts-api/commit/c9941c8b25189d75d2c02e8fd1256e1ead2611d7) does not create a reproducible build 

![diff](https://github.com/FoxxMD/multi-scrobbler/assets/4663766/9e7a71a1-3b42-4a97-8937-00d969c4b522)

and the source version **does not** work with FF cookies. I suspect the author published additional work in the NPM package that is not present in the repo.

Since the author has not yet [merged my fix for #86](https://github.com/nickp10/youtube-music-ts-api/pull/27) I cannot use the official build and since the source build does not work I cannot just update my [downstream fork with the included fix.](https://github.com/FoxxMD/youtube-music-ts-api/tree/libraryHistoryImprovements)

So for now everyone will need to use chrome to get cookies. If you'd like to get this to resolved you can [comment on my PR](https://github.com/nickp10/youtube-music-ts-api/pull/27) so the author will notice it?

**Comment by FoxxMD** [comment]
This is included in [0.6.0](https://github.com/FoxxMD/multi-scrobbler/releases/tag/0.6.0). I'll re-open this if there are still issues.
